src/vs/workbench/api/common/menusExtensionPoint.ts

```diff
@@ -53,6 +53,11 @@ const apiMenus: IAPIMenu[] = [
 		id: MenuId.EditorContext,
 		description: localize('menus.editorContext', "The editor context menu")
 	},
+	{
+		key: 'editor/context/copy',
+		id: MenuId.EditorContextCopy,
+		description: localize('menus.editorContextCopyAs', "'Copy as' submenu in the editor context menu")
+	},
```

Comment:
mark as proposed or finalize straight away?

Reply:
Up to API champ, I don't mind either way.